Cargo.toml

```diff
@@ -19,10 +19,10 @@ chrono = "0.4.40"
 tantivy = "0.24.0"
 itertools = "0.14.0"
 futures = "0.3.31"
-pythonize = "0.21.0"
+pythonize = "0.23.0"
 serde = "1.0"
 serde_json = "1.0.140"
 
 [dependencies.pyo3]
-version = "0.21.0"
+version = "0.23.5"
```

Comment:
Should we use the same version as pyo3 build config above 0.24.2?

Reply:
Just took care of this, and went ahead and `rebase`'d atop `master` too